src/tss2-rc/tss2_rc.c

```diff
@@ -0,0 +1,895 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
```

Comment:
Copyright tag missing

Reply:
You don't need the copyright tag, so if we don't add it we then don't have to maintain it.